stof/doctrine-extensions-bundle/1.2/etc/packages/stof_doctrine_extensions.yaml

```diff
@@ -0,0 +1,21 @@
+stof_doctrine_extensions:
+    default_locale: en_US
+
+    # Activate the extensions you want
+    orm:
+        default:
+            tree: true
+            timestampable: false # not needed: listeners are not enabled by default
```

Comment:
I'd suggest adding most common extensions enabled and comment them, and add a small comment line explaining that nothing is enabled by default, so users just have to uncomment the lines to activate a specific extension